.bit/responses/4.1-Week 4 Step 1.md

```diff
@@ -100,13 +100,26 @@ To learn more, read the docs [here](https://www.w3schools.com/jquery/html_text.a
 
 </details>
 
-## How do I call the event handler from my HTML form?
+## How do I trigger the function when the form submits?
 
-The `onclick` event executes a Javascript function when a button is clicked
+The event listener triggers the JavaScript function when a button is clicked.
+Here is how the form should look in the bunnimage index.html file:
 
 ```html
-  <label for="name">Code: </label> <br>
-  <input type="text" id="username" name="name" aria-label="name" /> <br>
-  <input value="Submit" type="button" onclick="getImage()" />
+<form id="bunnForm">
+   <label for="name">Code: </label> <br>
+   <input type="text" id="username" name="name" aria-label="name" /> <br>
+   <input value="Submit" type="button" />
+</form>
+```
+
+Here is the JavaScript:
+
+```JavaScript
```

Comment:
For this change to work the JS must be at the end of the file otherwise the items on the page will not have loaded and the event handler will not attach. Is it worth mentioning this?

Reply:
@dtetreau251 I think this is a good point to make ^^